### PR TITLE
feat(core): add readable media aliases

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/core/styles/variables/media.less [hunk 1]
```diff
@@ -21,3 +21,9 @@ next min-width - 60% from 1px (1/16 * 0.6 = 0.0375)
 @tui-desktop-interval: ~'(min-width: 64em) and (max-width: 79.9625em)';
 
 @tui-desktop-lg-min: ~'screen and (min-width: 80em)';
+
+@tui-touch: ~'(hover: none) and (pointer: coarse)';
+@tui-stylus: ~'(hover: none) and (pointer: fine)';
+@tui-pointer: ~'(hover: hover) and (pointer: coarse)';
+@tui-mouse: ~'(hover: hover) and (pointer: fine)';
+@tui-HD-color: ~'(dynamic-range: high) or (color-gamut: p3)';
```
Comment (waterplea) [inline projects/core/styles/variables/media.less:29]: I don't think we need this one :)

File: projects/core/styles/variables/media.scss [hunk 2]
```diff
@@ -16,3 +16,9 @@ $tui-desktop-min: 'screen and (min-width: 64em)';
 $tui-desktop-interval: '(min-width: 64em) and (max-width: 79.9625em)';
 
 $tui-desktop-lg-min: 'screen and (min-width: 80em)';
+
+$tui-touch: '(hover: none) and (pointer: coarse)';
+$tui-stylus: '(hover: none) and (pointer: fine)';
+$tui-pointer: '(hover) and (pointer: coarse)';
+$tui-mouse: '(hover) and (pointer: fine)';
+$tui-HD-color: '(dynamic-range: high) or (color-gamut: p3)';
```
Comment (waterplea) [inline projects/core/styles/variables/media.scss:24]: Same

File: projects/core/styles/variables/media.less [hunk 3]
```diff
@@ -21,3 +21,9 @@ next min-width - 60% from 1px (1/16 * 0.6 = 0.0375)
 @tui-desktop-interval: ~'(min-width: 64em) and (max-width: 79.9625em)';
 
 @tui-desktop-lg-min: ~'screen and (min-width: 80em)';
+
+@tui-touch: ~'(hover: none) and (pointer: coarse)';
+@tui-stylus: ~'(hover: none) and (pointer: fine)';
+@tui-pointer: ~'(hover: hover) and (pointer: coarse)';
+@tui-mouse: ~'(hover: hover) and (pointer: fine)';
```
Comment (waterplea) [inline projects/core/styles/variables/media.less:28]: I'm pretty sure we have some places in code where we only check for `(hover: hover)`. Let's replace them all with `@tui-mouse`, I don't think we ever need to have hover effects for coarse pointer.

**Review by waterplea — APPROVED** [review]

File: projects/core/styles/variables/media.scss (hunk 2 above)
Comment (splincode) [reply projects/core/styles/variables/media.scss]: fixed

File: projects/core/styles/variables/media.less (hunk 3 above)
Comment (splincode) [reply projects/core/styles/variables/media.less:28]: fixed

File: projects/core/styles/variables/media.less (hunk 1 above)
Comment (splincode) [reply projects/core/styles/variables/media.less]: fixed

File: projects/core/styles/mixins/wrapper.scss [hunk 4]
```diff
@@ -1,5 +1,7 @@
+@import '../variables/media.scss';
+
 @mixin wrapper-hover {
-    @media (hover: hover) {
+    @media ($tui-mobile) {
```
Comment (waterplea) [inline projects/core/styles/mixins/wrapper.scss:4]: Unnecessary brackets here and in some other places

Comment (splincode) [reply projects/core/styles/mixins/wrapper.scss:4]: But

image

image

Comment (vladimirpotekhin) [inline projects/core/styles/mixins/wrapper.scss:4]: tui-mouse ?

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/core/styles/mixins/wrapper.scss (hunk 4 above)
Comment (splincode) [reply projects/core/styles/mixins/wrapper.scss]: thank you

**Pull request merged**